src/board.py

```diff
@@ -81,3 +81,61 @@ def is_out_of_board(pawn):
 
 def is_out_of_base_line(coord):
     return coord < 0 or coord >= BASE_LINE_SIZE
+
+
+def is_crossable_right(pawn, fences):
+    top_right_fence = get_top_right_fence(pawn, fences)
```

Comment:
You should first test if the pawn is on the right side on the board if so return false.